components/Toast/react.stories.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { ToastContainer, toast } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
+import { mCheck } from '@masonite/svg-icons'
+import { Toast, toastContainerProps } from './react'
+import Button from '../Button/react'
+
+const Template = ({ includeButtons, ...props }) => {
+  const buttonsOverride = includeButtons
+    ? {
+      buttons: closeToast => ([
+        <Button key="close" onClick={closeToast}>
+          Close
+        </Button>,
+        <Button key="nothing" onClick={() => { return false }}>
+          Do Nothing
+        </Button>,
+      ])
+    } : null
+
+  const onClick = () => {
+    toast(
+      <Toast {...{ ...props, ...buttonsOverride }} />,
+      { autoClose: false, closeOnClick: false }
+    )
+  }
+
+  return <Button {...{ onClick }}>SHOW</Button>
```

Comment:
It is preferable to rely on CSS for custom casing (unless this is an acronym). There is an uppercase boolean prop on Button you can use.

Reply:
This was actually just a button to trigger the Storybook, but I made the change.